### DISCUSSION TITLE
为什么我的绳网这么多圈圈

### DISCUSSION THREAD

**Discussion by zzz1423** [share121/inter-knot#4633] [灌水]
难道我家网线被以太侵蚀了？我家邦布说网络是正常的，也能正常访问到空洞网站。
![PixPin_2024-09-08_20-10-12](https://github.com/user-attachments/assets/53080d17-4974-492e-b41b-e8c71baaa3a3)

**Comment by The-Place-Where-Winter-Sieeps** [comment]
额。。。可能。。。
你只是单纯的卡

**Comment by BuYaoHoYo** [comment]
帖子都是从Github上拉取的哦，建议上Github加速器

**Comment by wyc-26** [comment]
挂加速器试试，绳网上的图片都是在 GitHub 上存的

**Comment by HLXS-HLXS** [comment]
你只是单纯的卡了
